doozerlib/cli/__main__.py

```diff
@@ -874,8 +874,9 @@ def print_build_metrics(runtime):
 @click.option("--filter-by-os", default=None, metavar="ARCH",
               help="Specify an exact arch to push (golang name e.g. 'amd64').")
 @click.option('--dry-run', default=False, is_flag=True, help='Do not build anything, but only print build operations.')
+@click.option('--build-retries', default=1, help='Number of retries for an osbs build')
```

Comment:
@joepvd I believe `type=int` is needed here, otherwise we will get TypeError.

Reply:
Thanks! I think this is fixed in the new iteration